Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1

```diff
@@ -49,12 +49,39 @@ Function New-ARMParameterObject
         $CDBAccountName     = $Name
         $FunctionName       = $Name
         $FrontDoorName      = $Name
+        $appConfigName      = $Name
+        $aspGenevaName      = $Name
 
         ## The names of the Azure Cosmos Database and Container - Do not change (Must match with the values in the compiled
         ## Windows Package Manager Functions [WinGet.RestSource.Functions.zip])
         $CDBDatabaseName    = "WinGet"
         $CDBContainerName   = "Manifests"
 
+        ## The values required for Function ARM Template
```

Comment:
The ASP is for internal use - there should be a normal ASP that can be used externally

Reply:
Disregard this